src/main/scala/uk/gov/dhsc/htbhf/ClaimSimulation.scala

```diff
@@ -47,15 +51,25 @@ class ClaimSimulation extends Simulation {
     ninoFormat
   }
 
-  private def getRandomAlphabetCharAsString() = {
-    val randomChar = (random.nextInt(26) + 'A').asInstanceOf[Char]
+  private def getRandomNinoCharAsAString: String = {
+    val randomChar = VALID_NINO_CHARS.charAt(random.nextInt(VALID_NINO_CHARS.length))
     randomChar.toString
   }
 
+  private def getRandomValidTwoNinoCharsAsAString = {
+    var randomChars:String = ""
+
+    do {
+      randomChars = getRandomNinoCharAsAString + getRandomNinoCharAsAString
+    } while(INVALID_NINO_FIRST_TWO_CHARS contains randomChars)
+
+    randomChars
+  }
+
   val randomNinos = Iterator.continually(
     // Random number will be accessible in session under variable "OrderRef"
     Map("nino" -> {
-      getRandomAlphabetCharAsString + getRandomAlphabetCharAsString + ninoFormat.format(random.nextInt(999999)) + "D"
+      getRandomValidTwoNinoCharsAsAString + ninoFormat.format(random.nextInt(999999)) + "D"
```

Comment:
I'm guessing we don't care what the first two chars of the NINO are for the performance tests?

Reply:
I don't think the performance tests check what the response contains

Reply:
Yep, we've always been generating the nino completely randomly which will give a mix of active and rejected claims.